azkaban-common/src/main/java/azkaban/jobExecutor/JavaProcessJob.java

```diff
@@ -147,4 +148,13 @@ protected String createArguments(List<String> arguments, String separator) {
 
     return "";
   }
-}
+
+  protected Pair<Long, Long> getProcMemoryRequirement() {
```

Comment:
Maybe add quick a comment on what this method is doing and how is it computing the ProcMemory?   It doesn't seem like a trivial method.